Runtime/Scripts/EdgeEventsConfig.cs

```diff
@@ -0,0 +1,74 @@
+﻿using System;
+using UnityEngine;
+namespace MobiledgeX
+{
+    [Serializable]
+    public struct EdgeEventsConfig
+    {
+        /// <summary>
+        /// Latency threshold (In Milliseconds) as the limit to automatically migrate to a new cloudlet
+        /// </summary>
+        [Tooltip("Latency threshold (In Milliseconds) as the limit to automatically migrate to a new cloudlet")]
+        public double latencyThresholdTriggerMs;
+        /// <summary>
+        /// Port information for latency testing
+        /// <para>Use 0 to select a random port</para>
+        /// </summary>
+        [Tooltip("Port information for latency testing" +
+            "\nUse 0 to select a random port")]
+        public int latencyTestPort;
+        /// <summary>
+        /// Config for latency updates
+        /// </summary>
+        [Tooltip("Config for latency updates")]
+        public ClientEventsConfig latencyConfig;
+        /// <summary>
+        /// Config for location updates
+        /// </summary>
+        [Tooltip("Config for location updates")]
+        public ClientEventsConfig locationConfig;
+        //fixme newFindCloudletEventTriggers
+    }
+
+    [Serializable]
+    public struct ClientEventsConfig
+    {
+        /// <summary>
+        /// UpdatePattern for sending client events
```

Comment:
It looks like a lot of these parameter descriptions are also used in MobiledgeXEditorWindow.cs. Is it possible to consolidate these? If not, no worries.

Reply:
They are different. One is code comment and the other is just Editor Hover Tooltips for the Editor Window.